src/legacy/ui/public/chrome/directives/header_global_nav/components/header_badge.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EuiBetaBadge } from '@elastic/eui';
+import React, { Component } from 'react';
+import * as Rx from 'rxjs';
+
+interface Props {
+  badge$: Rx.Observable<ChromeBadge | undefined>;
+}
+
+interface State {
+  badge: ChromeBadge | undefined;
+}
+
+import { ChromeBadge } from '../../../../../../../core/public';
+
+export class HeaderBadge extends Component<Props, State> {
+  private subscription?: Rx.Subscription;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = { badge: undefined };
+  }
+
+  public componentDidMount() {
+    this.subscribe();
+  }
+
+  public componentDidUpdate(prevProps: Props) {
+    if (prevProps.badge$ === this.props.badge$) {
+      return;
+    }
+
+    this.unsubscribe();
+    this.subscribe();
+  }
+
+  public componentWillUnmount() {
+    this.unsubscribe();
+  }
+
+  public render() {
+    if (this.state.badge == null) {
+      return null;
+    }
+
+    return (
+      <div
+        className="chrHeaderBadge__wrapper"
+        data-test-subj="headerBadge"
+        aria-label={this.state.badge.text}
```

Comment:
This aria-label is no longer needed since the BetaBadge connects the aria attributes with the tooltip.

<img width="609" alt="Screen Shot 2019-04-12 at 10 39 15 AM" src="https://user-images.githubusercontent.com/549577/56045497-598fce00-5d0f-11e9-9a23-742be51aae16.png">

Reply:
I'm only seeing the euiToolTipPopover in the DOM when hovering over the tooltip. I primarily added this for use within the [global nav functional testing service](https://github.com/elastic/kibana/pull/34551/files#diff-5841023215c87b8fa2d1ff1fd103ba65R49). If we're confident that screenreaders behave properly without this `aria-label` itself, I can switch to something like `data-test-badgeText`.

Reply:
It also shows in the DOM on focus so when users tab to the item, it will appear:

<img src="https://d.pr/free/i/xhvSAR+" width="50%" />